internal/providers/bedrock_mantle.go

```
@@ -0,0 +1,356 @@
+	}
+}
+
+func (b *BedrockMantleProxy) UserIDFromRequest(_ *http.Request) string { return "" }
+
+// ValidateAPIKey accepts only normal llm-proxy keys registered for Mantle.
+// The resolved provider credential is deliberately discarded: Mantle is
+// authenticated solely by the local AWS SigV4 signature.
+func (b *BedrockMantleProxy) ValidateAPIKey(req *http.Request, keyStore APIKeyStore) error {
+	auth := req.Header.Get("Authorization")
+	if !strings.HasPrefix(auth, "Bearer ") {
+		return fmt.Errorf("Bearer proxy API key is required")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)